src/mumble/UserModel.cpp

```diff
@@ -1507,3 +1515,4 @@ void UserModel::updateOverlay() const {
 	g.o->updateOverlay();
 	g.lcd->updateUserView();
 }
+
```

Comment:
Superfluous newline at the end? I see no newline complaint in the diff view here, so I assume there is now two newlines at the end?